include/openssl/dh.h

```diff
@@ -385,4 +385,8 @@ BSSL_NAMESPACE_END
 #define DH_R_INVALID_NID 106
 #define DH_R_INVALID_PARAMETERS 107
 
+// error code for nodejs compatibility
```

Comment:
nit: we might want to remove this comment. other projects we integrate with may depend on this as well, nodejs just happens to be the first one.